src/.travis.yml

```
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```

```suggestion
    - $HOME/.m2
```
here same situation with empty line